src/helpers/constants.js

```diff
@@ -471,4 +471,17 @@ module.exports = {
   ACTIVATED: 'Actif',
   STOPPED: 'Arrêté',
   ARCHIVED: 'Archivé',
+  // DURATION
+  get DURATION_UNITS() {
```

Comment:
Qu'est-ce qui t'as fait choisir de le définir comme une methode plutot qu'un tableau ?

Reply:
j'ai l'impression que tout les tableaux ont cette tete dans le fichoier `constants.js` non ?

Reply:
Enfait il me semble qu'on utilise une methode uniquement si on a besoin du `this`. Mais peut etre qu'ici ca vaudrait le coup de declarer aussi les unité en constante et donc d'utiliser le this. Qu'en pensez-vous ?

Reply:
Ok j'ajoute les constantes alors